commit c680934c282c791f151f4efe173c53604bac7c93 (refs/changes/20/3820/2)
Author: Jason King <jason.king@joyent.com>
Date:   2018-04-16T13:33:56-05:00 (1 year, 6 months ago)
    
    OS-6891 usr/src/lib/fm is missing a dependency on libpcidb
    Reviewed by: Patrick Mooney <pmooney@pfmooney.com>
    Approved by: Mike Gerdts <mike.gerdts@joyent.com>

diff --git a/usr/src/lib/Makefile b/usr/src/lib/Makefile
index 445238f973..7d337e7175 100644
--- a/usr/src/lib/Makefile
+++ b/usr/src/lib/Makefile
@@ -588,7 +588,8 @@ auditd_plugins: libbsm libsecdb libgss libmtmalloc
 brand:		libzonecfg libmapmalloc libipadm libcmdutils libproc librpcsvc
 cfgadm_plugins:	libdevice libdevinfo libhotplug librcm hbaapi libkstat libscf
 fm:		libexacct libipmi libzfs scsi libdevinfo libdevid libcfgadm \
-		libcontract libsysevent ../cmd/sgs/libelf libdladm libsff
+		libcontract libsysevent ../cmd/sgs/libelf libdladm libsff \
+		libpcidb
 $(SPARC_BLD)fm:	libpri
 gss_mechs/mech_dh: libgss
 gss_mechs/mech_dummy: libgss
